ccc/elasticsearch.py

```diff
@@ -115,3 +117,31 @@ def store_document(
             *args,
             **kwargs,
         )
+
+    def store_bulk(
+        self,
+        body: str,
+        inject_metadata=True,
+        *args,
+        **kwargs,
+    ):
+        util.check_type(body, str)
+
+
+        if inject_meta and _metadata_dict():
```

Comment:
according to linter, this is a typo (-> inject_metadata). In case you copied that from my proposed code snippet: sry